### ISSUE TITLE
autocomplete suggestions interpolation bug in version 1.0-rc1 or olders

### ISSUE THREAD

**Issue by nestorneto** [angular/material#5394]
![ezgif com-optimize](https://cloud.githubusercontent.com/assets/362382/10773118/5cf73666-7cd8-11e5-9a3a-1a3e2339d2d3.gif)

at the first time you open autocomplete suggestions the text before interpolation appear in the blink of the eye. in old computers is more visible.

**Comment by hanford** [comment]
+1

**Comment by topherfangio** [comment]
@robertmesserle We discussed this issue a while ago and I seem to remember you wanted to look at it and/or thought you had a solution. Did you ever get the chance to dig in and take a look?

**Comment by robertmesserle** [comment]
@topherfangio I believe this is a duplicate of another issue.  I looked into this and my initial guess was wrong - it seems to be related to virtual repeater, and @kseamon is looking into it.

**Comment by robertmesserle** [comment]
@kseamon I tried removing all timeouts within the parent scope directive, but this still flickers.  (Although the flicker is very quick.)  You said you had a version that had no flicker, right?

**Comment by kseamon** [comment]
This is a duplicate of #5188 and will be fixed once #5637 is merged.

**Comment by robertmesserle** [comment]
@kseamon Ah, awesome.  Closing as duplicate.
